test/test_commit.py

```diff
@@ -509,3 +509,14 @@ def test_trailers(self):
         assert KEY_1 not in commit.trailers.keys()
         assert KEY_2 in commit.trailers.keys()
         assert commit.trailers[KEY_2] == VALUE_2
+
+    def test_commit_co_authors(self):
+        commit = copy.copy(self.rorepo.commit("4251bd5"))
+        commit.message = """Commit message
+
+Co-authored-by: Test User 1 <602352+test@users.noreply.github.com>
+Co-authored-by: test_user_2 <another_user-email@.github.com>"""
```

Comment:
Especially when `Actor` code is used for parsing it would be interesting to see what happens if there is a malformed one in the list. Will that be ignored, or will the whole operation fail?

Reply:
It should be ignored, as the regex will simply not find it, expanded the test to show this. 😄